src/EnumSet.php

```diff
@@ -170,7 +162,7 @@ public function next()
      */
     public function rewind()
     {
-        if ($this->bitset) {
+        if ($this->count() !==0) {
```

Comment:
The count operation is very expensive for this. Alternative we could use this (not tested): `$this->bitset === str_repeat("\0", ceil($this->ordinalMax / 8))` or `trim($this->bitset, "\0") === ''`